<launch>
    <!-- Autogenerated launch file - do not hand edit -->
    <include file="{{machine_file}}" />

    <group ns="transform_2d_calibrator_params">
        <rosparam command="load" file="{{params_file}}"/>
    </group>
    {% for namespace, image_topic_0, image_topic_1 in launch_list %}
    <group ns="{{namespace}}">
        <node pkg="mct_transform_2d" type="transform_2d_calibrator.py" name="transform_2d_calibrator" args="{{image_topic_0}} {{image_topic_1}}" machine="mct_master" />
    </group>
    {% endfor %} 
</launch>
